api/CanMakePaymentEvent.json

```diff
@@ -6,7 +6,7 @@
         "spec_url": "https://w3c.github.io/payment-handler/#the-canmakepaymentevent",
         "support": {
           "chrome": {
-            "version_added": "70"
+            "version_added": "68"
```

Comment:
version 70 comes from the collector per https://github.com/mdn/browser-compat-data/pull/13309. Who is correct? cc @foolip

Reply:
I got 68 from https://chromestatus.com/feature/5160285237149696 but as we discussed last week, this could be wrong. Let me know if you'd rather me put the data for these items back to 70.

Reply:
OK, erroneous 68s reverted back to 70.

Reply:
When there's a disagreement between the collector and some other data source I would suggest looking at the test and asking if there's any way it could give the wrong result. Could there have been a rename? An interface object not exposed? Does the test depend on something else that it shouldn't?

Once you're convinced the test is good, trust the results over chromestatus, which is updated by hand and not test results.